src/Aspire.Hosting/Dcp/DcpExecutor.cs

```diff
@@ -1777,6 +1775,10 @@ await execution.ExecuteAsync(async (attemptCancellationToken) =>
                 }, cancellationToken).ConfigureAwait(false);
             }
 
+            // Raise event after resource has been deleted. This is required the event sets the status to "Starting" and resources being
+            // deleted will temporarily override the status to a terminal state.
```

Comment:
```suggestion
            // Raise event after resource has been deleted. This is required because the event sets the status to "Starting" and resources being
            // deleted will temporarily override the status to a terminal state, such as "Exited".
```